src/components/UnlockWallet/index.tsx

```diff
@@ -67,7 +68,30 @@ const PrivacySection = (
 );
 
 export const UnlockWallet: React.FC<{}> = props => {
-  const { activate, error } = useWeb3React();
+  const { connector, activate, error } = useWeb3React();
+
+  const isMetamask = connector instanceof InjectedConnector;
+  const availableChains = useMemo(
+    () =>
+      Object.entries(internalAddressesPerNetwork).map(([name, addrs]) =>
+        isMetamask ? (
+          <Button
```

Comment:
The same advice from prior review comments regarding adding a `key` for array items, but you'll need to add it to `Button` as well as `Text` since you're branching here.

Alternatively, you can put `Button` and `Text` under a `React.Fragment`, and give *it* a key by wrapping it with something like the following:
```
<key={id}>
</>
```

Reply:
Did this myself in my commit, check it out and resolve this comment when you see it.